docs/usage/ruby.md

```diff
@@ -15,6 +15,12 @@ Renovate supports upgrading dependencies in Bundler's Gemfiles and their accompa
 1. A PR is created which updates the `Gemfile` and `Gemfile.lock` in a single commit
 1. If the source repository has a "changelog" file or uses GitHub releases, then Release Notes for each version are embedded in the generated PR
 
+## Caveats
+
+- Renovate doesn't update dependencies without a version constraint. Example: `gem 'some-gem', '~> 1.2.3'` will update
+  `some-gem` if a new version matching the constraint if available, but `gem 'some-gem'` won't. If you always want to
+  stay at the latest available version, consider specifying `gem 'some-gem', '> 0'`.
```

Comment:
```suggestion
Renovate doesn't update dependencies without a version constraint.
Example: `gem 'some-gem', '~> 1.2.3'` will update `some-gem` if a new version matching the constraint is available, but `gem 'some-gem'` won't.
If you always want to have the latest available version, consider specifying `gem 'some-gem', '> 0'`.
```

I think a normal paragraph looks better here.
We use one sentence per line in paragraph text, so I've gone ahead and made that change.
Also fixed a typo: `if available` -> `is available`.

Reply:
Thanks for the suggestion!

The reason I used a list here is that I anticipated future additions here where somebody might add more caveats => if it's already formatted as a list, it doesn't mark the (presumably unchanged) line in the diff – much like adding commas in the last key-value pair in multiline JavaScript objects. But I guess you could fairly argue YAGNI and even if it turns out it's needed, it's not a huge issue. :)